requirements.txt

```diff
@@ -0,0 +1,7 @@
+pytest~=6.2.5
```

Comment:
resqpy dependencies should be handled via existing configuration files – what is this file for?